packages/server/src/app-server.ts

```diff
@@ -2,19 +2,21 @@ import { timeout, spawn } from 'effection';
 import { fetch } from '@effection/fetch';
 import { exec, Process } from '@effection/node';
 import * as process from 'process';
-import { OrchestratorState, AppOptions, Service, AppServiceStatus } from './orchestrator/state';
+import { OrchestratorState, AppOptions, Service, AppServiceStatus, ServiceState } from './orchestrator/state';
 import { Slice } from '@bigtest/atom';
 import { restartable } from './effection/restartable'
 import { assert } from 'assert-ts';
 
-export const appServer: Service<AppServiceStatus, AppOptions> = (slice, options) => {
-  let appOptions = options.atom.slice('appService', 'options');
+export const appServer: Service<AppServiceStatus, AppOptions> = (serviceSlice) => {
+  let appOptions = serviceSlice.slice('options');
 
-  return restartable(appOptions, startApp(slice));
+  return restartable(appOptions, startApp(serviceSlice));
 }
 
-const startApp = (appStatus: Slice<AppServiceStatus, OrchestratorState>) => function* (options: AppOptions) {
+const startApp = (serviceSlice: Slice<ServiceState<AppServiceStatus, AppOptions>, OrchestratorState>) => function* (options: AppOptions) {
   assert(!!options.url, 'no app url given');
+
+  let AppServiceStatus = serviceSlice.slice('status')
```

Comment:
Should this be `appServiceStatus` with a lower case first letter?

Reply:
@jnicklas absolutely.  Good spot